tests/unit/utils/test_fs.py

```diff
@@ -72,18 +75,17 @@ def test_should_return_false_on_no_more_dirs_below_path(
 
     @patch.object(System, "is_symlink", return_value=False)
     def test_should_call_recursive_on_no_condition_matched(self, _):
```

Comment:
This test can also be refactored. Just extract it to standalone test, not part of `TestContainsLink` class.

Reply:
Done!